backend/src/bundles/users/user.service.ts

```diff
@@ -19,6 +19,11 @@ class UserService implements Service {
         return Promise.resolve(null);
     }
 
+    public async findById(id: number): Promise<UserFindResponseDto> {
+        const user = await this.userRepository.findById(id);
```

Comment:
Looks like we can make it like this
```suggestion
        const user = await this.userRepository.find({ id });
```

```suggestion
        const user = await this.userRepository.find({ email });
```

Reply:
In this case we don't need in the interface `findById` method. Because `find(payload)` cover all these things

Reply:
more examples
```js
    private async find(payload: object): Promise<UserEntity | undefined> {
        return await this.userRepository.find(payload);
    }

    public async findByEmail(email: string): Promise<UserEntity | undefined> {
        return await this.find({ email });
    }

    public async findById(id: string): Promise<UserEntity | undefined> {
        return await this.find({ id });
    }
```

Reply:
Done